content/en/blog/contributor-contest/index.md

```diff
@@ -0,0 +1,128 @@
+2. Identify defects, raise issues in respective repos [kubeedge](https://github.com/kubeedge/kubeedge) / [beehive](https://github.com/kubeedge/beehive) / [viaduct](https://github.com/kubeedge/viaduct) / [website](https://github.com/kubeedge/website).
+3. Re-solving existing issues in repos [kubeedge](https://github.com/kubeedge/kubeedge) / [beehive](https://github.com/kubeedge/beehive) / [viaduct](https://github.com/kubeedge/viaduct) / [website](https://github.com/kubeedge/website).
+4. Share requirements by creating issues in repo [kubeedge](https://github.com/kubeedge/kubeedge).
+5. Writing blogs about KubeEdge either in the [KubeEdge website](https://kubeedge.io) (on PR approved & merged, this gets published in kubeedge.io website blog) or in other technical blogging site. Please refer here to know how to write a [KubeEdge blog](https://github.com/kubeedge/website/wiki/How-to-write-a-blog).
+Submit your blog details  [here](https://docs.google.com/forms/d/e/1FAIpQLSd9luGaHWt5jABEQjtr11CgWLVw7Px6tJf0TIjidbl3UL_Izg/viewform).
+6. Create your own sample applications and demo examples to illustrate possible use case(s) of using KubeEdge in repo [examples](https://github.com/kubeedge/examples).
+
+{{% /alert %}}
+
+## **Who can participate ?**
+
+At the moment most of the contributors are from Huawei organization. We want to encourage contributions from individual contributors or individuals of other organizations.  
+Contributions made from accounts linked to email ids of huawei organization will not be eligible for participation. 
+The contest is open to all other members.
+
+
+## **How the winners are selected ?**
+
+Any code contributions should follow the [contribution flow](https://github.com/kubeedge/kubeedge/blob/master/CONTRIBUTING.md) to get accepted. We consider the code submitted along with PR(s) for feature / test case development or issue fix. For issues created, will check severity of issue, quality of steps described towards reproducing, data sufficiency. If a requirement is logged as issue, then quality of description, uniqueness & how much value it could create in comparison with other edge computing platforms taken into consideration.  
+
+For a blog written, we consider content & popularity of that blog.  
+
+For an example created, we consider code, documentation describing the steps & user guide.  
+
+Considering above points, 3 winners will be selected.
+
+
+## **Timeline**
+
+{{% alert note %}}
+
+ Starts from **23rd April 2019 00:00 (UTC)** and ends on **22nd May 2019 23:59 (UTC)**
+
+{{% /alert %}}
+
+
+## **How the winners are notified ?**
+
+{{% alert note %}}
+
+On **23rd May 2019 00:00 (UTC)** through e-mail, slack, twitter and results announcement blog will be published. 
```

Comment:
We will make the winner announcement blog on **23rd May 2019 00:00 (UTC)** via e-mail, slack, wechat, twitter.